ssh_filter_btrbk.sh

```diff
@@ -53,6 +53,12 @@ run_cmd()
 
 reject_filtered_cmd()
 {
+    # reject commands that contain a newline (LF character)
+    local number_of_newlines=""; number_of_newlines="$(printf '%s' "$SSH_ORIGINAL_COMMAND" | wc -l)"
```

Comment:
Yes, I have it removed already in my local branch... just not pushed yet.

As said... please wait with any merges of this branch here - there's still a lot to do, and I think it would get messier.